### ISSUE TITLE
Translate date in "printable schedule"

### ISSUE THREAD

**Issue by cubizh** [fw42/cubecomp#189]
Add the date printed in the printable schedule (in backend Event page) to the translation list to be translated.

**Comment by fw42** [comment]
Is this really necessary? Do event organizers not want to generally print those things in english? In my experience, even at german competitions, those things usually happen in english since there is always some people who only speak english but not german (or portuguese in your case).

**Comment by SAuroux** [comment]
I agree with you there. Even with this option I would still use the English
version as that's the one that likely every competitor understands.
Am 11.07.2015 02:25 schrieb "Florian Weingarten" <notifications@github.com>:

**Comment by cubizh** [comment]
To me, it's the completely opposite view. In our competitions, if I put things out for the public in english only, I'd get immediate complaints that people wouldn't understand (especially parents) and it looked careless on my part.
I think a better more professional and suited solution is to be able to choose to have all the information in any of the languages you defined in the settings, under "Available Languages" that your competition will have. That information is already present, all that's necessary is add words to be translated and add the proper links for the languages in the backend, like "Schedule (de)(en)(pt)(etc...)".

**Comment by fw42** [comment]
fair enough
